### PR TITLE
Adding status color + fix rgbaToCSS

### LINKED ISSUE #926: Add function to show status colour of glyphs in canvas
It'd be great to have a function to toggle show/hide the status colour (of the default layer) of each glyph in the canvas, maybe as a background colour of each glyph, or a coloured bar at the bottom.

This helps to have an overview of the status of a specific set of glyphs that we input in the text box.

### PR DESCRIPTION
**Pull request by ollimeier**
Fixes #926 

I am very sorry for the confusion.
Base on the feedback stored here #1342

### CODE REVIEW

**Comment by justvanrossum** [comment]
No problem! Sometimes git seems to be on a mission to ruin your day.

File: src/fontra/views/editor/visualization-layer-definitions.js [hunk 1]
```diff
@@ -1020,6 +1022,55 @@ registerVisualizationLayerDefinition({
...
+      positionedGlyph.varGlyph.sources[sourceIndex].customData[
+        "fontra.development.status"
+      ];
+    if (status === undefined) {
+      return;
+    }
+
+    const color = [...statusFieldDefinitions[status].color];
+    if (positionedGlyph.isEditing) {
+      // in editing mode make opacity 50%
```
Comment (justvanrossum) [inline src/fontra/views/editor/visualization-layer-definitions.js:1058]:
```suggestion
      // in editing mode reduce opacity
```

Comment (ollimeier) [reply src/fontra/views/editor/visualization-layer-definitions.js]: Thanks. done.

File: src/fontra/client/core/utils.js [hunk 2]
```diff
@@ -345,9 +345,10 @@ export function arrayExtend(thisArray, itemsArray) {
...
 export function rgbaToCSS(rgba) {
-  const channels = rgba.map((channel) => Math.round(channel * 255));
-  if (channels[3] == 255) {
-    channels.pop();
+  const channels = rgba.slice(0, 3).map((channel) => Math.round(channel * 255));
+  const alpha = rgba[3];
+  if (alpha !== undefined && 0 <= alpha && alpha < 1) {
+    channels.push(alpha);
   }
   return `${channels.length === 4 ? "rgba" : "rgb"}(${channels.join(",")})`;
```
Comment (justvanrossum) [inline src/fontra/client/core/utils.js:353]: I just read https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#syntax and it says:
```
The rgba() function can also be used to express sRGB colors. This is an alias for rgb() that accepts the same parameters.
```
That means we can simplify to:
```suggestion
  return `rgb(${channels.join(",")})`;
```
Would be a nice cleanup while you're at it.

**Comment by justvanrossum** [comment]
Thank you!!

**Pull request merged**